src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -71,6 +71,7 @@ public class KNNConstants {
     public static final VectorDataType DEFAULT_VECTOR_DATA_TYPE_FIELD = VectorDataType.FLOAT;
 
     public static final String RADIAL_SEARCH_KEY = "radial_search";
+    public static final String QUANTIZATION_STATE_FILE_SUFFIX = "qs";
```

Comment:
should we add some plugin extension too here. I feel qs being so generic what will happen if Lucene has this file extension at some point of time. 

@jmazanec15 , @Vikasht34 thoughts?

Reply:
agree - maybe qframe or pluginqs

Reply:
qstate?